functions/closestPair.js

```diff
@@ -1,9 +1,31 @@
 export default function closestPair(points) {
-  let result = {pair: [], distance: 99999}
+  const maxDistance = 9999999
```

Comment:
Since this is a constant that is independent of this function, I would declare it outside of the function, using CAPITAL_SNAKE_CASE:
```js
const MAX_DISTANCE = 9999999
```
Also, you might want to consider using the value [`Infinity`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Infinity) to make sure that it will always be larger than whatever you are comparing against (in which case you would not need the constant declaration, you could just use `Infinity` directly).